docs/zkapps/tutorials/03-deploying-to-a-network.mdx

```diff
@@ -1,32 +1,456 @@
+
+ We are now setting the active instance to the remote Berkeley network. We are connecting to Berkeley through a GraphQL proxy, which is running a Mina node connected to the Berkeley network. By connecting to Berkeley, we can provide smart contracts that are globally accessible, and provide strong guarantees around state due to both Mina's decentralization and its succinct state proof.
+
+With the Ethereum Bridge (to be included in a future tutorial), states and proofs from smart contracts deployed on Mina networks can also be available on Ethereum and other EVM chains.
+
+ We also set a transaction fee, which we'll use to pay for access to sending transactions and deploying smart contracts on Mina. Transaction fees in code are inputted as nano mina. We will use a 0.1 Mina fee as default (100,000,000 nano mina) here.
+
+:::note
+
+While in this case we are connecting to a remote RPC (run by `minaexplorer.com`), you can also run a Mina node locally, and instead use its GraphQL endpoint. While in other blockchains this would be very heavyweight, because Mina is succinct this is actually a reasonable option. See [here](/node-operators/getting-started) to see how to do this.
+
+A version of the Mina node is in the works that runs directly in the browser and within the local node process, so client side users can connect directly to Mina with low resource requirements, while keeping full decentralization guarantees, with no intermediaries that can go down, censor, or otherwise impact you or your users' connection to the network.
+
+:::
+
+Now, to finish setting up our code, let's add to our `main.ts`:
+```ts
+...
+  7 } from 'snarkyjs';
+  8
+  9 import fs from 'fs';
+ 10
+ 11 (async function main() {
+...
+ 23   let transactionFee = 100_000_000;
+ 24
+ 25   const configName = process.argv[2];
+ 26
+ 27   const deployerKeysFileContents = fs.readFileSync(
+ 28     'keys/' + configName + '.json',
+ 29     'utf8'
+ 30   );
+ 31
+ 32   const deployerPrivateKeyBase58 = JSON.parse(
+ 33     deployerKeysFileContents
+ 34   ).privateKey;
+ 35
+ 36   const deployerPrivateKey = PrivateKey.fromBase58(deployerPrivateKeyBase58);
+ 37
+ 38   const zkAppPrivateKey = deployerPrivateKey;
+ 39
+ 40   // ----------------------------------------------------
+...
+```
+
+This uses the key generated by the `zk config` tool, stored in `keys/`. The name of the key file will be provided through an argument on the command line (`process.argv[2]`). 
+
+You can run this now with:
+
+```sh
+npm run build && node build/src/main.js berkeley
+```
+
+Which will read keys from `keys/berkeley.json`. Public and Private Keys in Mina are commonly stored in `Base58` for easily readability. In Mina, public keys start with `B62`, and private keys start with `EKE` for easy differentiability.
+
+Our Smart Contract is also stored on chain in the same account we deployed from. So we set `zkAppPrivateKey = deployerPrivateKey`.
```

Comment:
> Our Smart Contract is also stored on chain in the same account we deployed from

Problematic b/c the smart contract isn't stored on chain. 

Could this sentence be excluded? If not, maybe something along the lines of: "Our zkApp account address is equal to the public key for the key pair that we created when using the `zk config` command." Idk, would probably just exclude it